step_test.rb

```diff
@@ -82,3 +82,8 @@
     puts "Collector token #{plist['APM_COLLECTOR_TOKEN']} in plist does not match test token #{ENV['APM_COLLECTOR_TOKEN']}"
     exit 1
 end
+
+if plist['APM_INSTALLATION_SOURCE'] != "Trace step"
+    puts "Collector token #{plist['APM_COLLECTOR_TOKEN']} in plist does not match test token #{ENV['APM_COLLECTOR_TOKEN']}"
```

Comment:
Has the same message with the if statement above. Shouldn't it have a different one, something like "The installation source is not the Trace SDK..."? Or if it should have the same behaviour, you could consider merging the two if statements with an "OR".

Reply:
ill update the message now.